### ISSUE TITLE
confused at detach()

### ISSUE THREAD

**Issue by SnallQiu** [uvipen/Super-mario-bros-PPO-pytorch#5]
hello, thanks for your code.
I have read your ppo code carefully, and i'm really confused at one function : detach() 
`old_log_policies = torch.cat(old_log_policies).detach()`
or 
`values = torch.cat(values).detach()`

i understand the detach() function is to stop down the autograd backward
but still wondering why you "detach" them in your code ?
